modules/bibupload/lib/batchuploader_webinterface.py

```diff
@@ -48,8 +53,10 @@
 class WebInterfaceBatchUploaderPages(WebInterfaceDirectory):
     """Defines the set of /batchuploader pages."""
 
-    _exports = ['', 'metadata', 'metasubmit', 'history', 'documents',
-        'docsubmit', 'daemon', 'allocaterecord', 'confirm']
+    _exports = [
+        '', 'metadata', 'metasubmit', 'history', 'documents', 'docsubmit',
+        'daemon', 'allocaterecord', 'allocaterecordandtagasmigrated', 'confirm'
```

Comment:
you would need to change the name here as well

Reply:
should I remove the `allocaterecord ` or not?

Reply:
nop, I think it is still used by other services.